app/Http/Controllers/Api/UnsubscribeController.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Gladiator\Http\Controllers\Api;
+
+use Gladiator\Services\Manager;
+use Gladiator\Models\Competition;
+use Gladiator\Http\Requests\UnsubscribeRequest;
+
+class UnsubscribeController extends ApiController
+{
+    /**
+     * @var Gladiator\Services\Manager;
+     */
+    protected $manager;
+
+    public function unsubscribe(UnsubscribeRequest $request)
+    {
+        $credentials = $request->all();
```

Comment:
I think it would be more readable to name this like `$data` or something. "credentials" sound like you are expecting a username/password to be sent in the request.